README.md

```diff
@@ -36,13 +36,10 @@ $ npm i --save hydra-js
 var Hydra = require('hydra-js')
 
 const config = {
- client: {
-   id: process.env.HYDRA_CLIENT_ID, // id of the client you want to use, defaults to this env var
-   secret: process.env.HYDRA_CLIENT_SECRET, // secret of the client you want to use, defaults to this env var
- },
- auth: {
-   tokenHost: process.env.HYDRA_URL, // hydra url, defaults to this env var
- }
+ clientId:     process.env.HYDRA_CLIENT_ID,     // id of the client you want to use, defaults to this env var
```

Comment:
Thank you for the changes! However, since this change here is only appealing to the eye I would like to keep the structure as is and prevent breaking changes! Also, it's closer to the simple-oauth2 config.